sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileBasedSink.java

```diff
@@ -791,7 +791,7 @@ final void removeTemporaryFiles(
                   FileSystems.match(Collections.singletonList(tempDir.toString() + "*")));
           for (Metadata matchResult : singleMatch.metadata()) {
             if (allMatches.add(matchResult.resourceId())) {
-              LOG.info("Will also remove unknown temporary file {}", matchResult.resourceId());
+              LOG.warn("Will also remove unknown temporary file {}", matchResult.resourceId());
```

Comment:
Could we add some more info here. Something like:
- "Make sure that no other process/job is using the same temporary folder. This might result in data consistency issues"

or maybe a link to a better answer somewhere else like a JIRA etc.

Reply:
Thanks for the comment. Updated the log message.